src/components/spec-document/overview/ServerVariables.vue

```diff
@@ -0,0 +1,96 @@
+<template>
+  <div
+    v-if="server.variables"
+    class="variables-container"
+  >
+    <span class="variables-label">Variables</span>
+    <div
+      v-for="key in Object.keys(server.variables)"
```

Comment:
Since we need both key and value during each iteration, it might be easier this way

```suggestion
      v-for="(value, key) in server.variables"
```